server/authMiddleware/jwt.js

```diff
@@ -0,0 +1,30 @@
+import jwt from 'jsonwebtoken';
+// if (typeof checkTokenResult !== 'boolean') {
+//   return res.send(500).send({
+//     message: 'Sever Error occured during jwt verification',
+//     error: checkTokenResult
+//   });
+// }
+// const checkToken = (token) => {
+//   const { id } = jwt.verify(token, process.env.SECRET_KEY);
+//   return query('SELECT ID FROM USERS WHERE id = $1', [id])
+//     .then(result => typeof result.row[0].id === 'string')
+//     .catch(error => error);
+// };
+const verifyToken = (req, res, next) => {
```

Comment:
It's cleaner if you have `request` instead of `req`, `response` instead of `res`.